operator/internal/controller/redpanda/redpanda_controller.go

```diff
@@ -419,6 +415,10 @@ func (r *RedpandaReconciler) reconcileDefluxed(ctx context.Context, rp *redpanda
 		}] = struct{}{}
 	}
 
+	if len(errs) > 0 {
```

Comment:
Can we add a release note to indicate the behavior change here?

Reply:
Yes, I will add a note.